internal/vault/cache_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	lru "github.com/hashicorp/golang-lru"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_clientCache_Prune(t *testing.T) {
+	var (
+		dummyCallbackFunc = func(key, value interface{}) {}
+		size              = 256
+		cacheLen          = 1
+		err               error
+		ctrl              = gomock.NewController(t)
```

Comment:
I wonder if we cold move these variable to become members of the test struct below. That way we could possible modify them for each test.